### PR TITLE
toggle checks

### PR DESCRIPTION
**Pull request by shinriyo**
it was forgotten feature. so added.

### CODE REVIEW

**Comment by shinriyo** [comment]
travis-ci is good worker!
I just code in GitHub, so I forgot. I'm sorry...

File: examples/todoapp/todoapp.nim [hunk 1]
```diff
@@ -64,6 +64,12 @@ proc onAllDone(ev: Event; n: VNode) =
   clear()
   selectedEntry = -1
 
+proc toggleAll(ev: Event; n: VNode) =
+  var notes = document.getElementsByClassName("toggle")
```
Comment (Araq) [inline examples/todoapp/todoapp.nim]: This is not how to do things. It needs to update the data model. Karax will then update the DOM.

Comment (Araq) [reply examples/todoapp/todoapp.nim]: What is the "DOM" program? Please read this https://nim-lang.org/araq/karax.html

**Comment by Araq** [comment]
"Toggle all" works in the todoapp already. Closing.

**Pull request closed without merging**